modules/drivers/redshift/src/metabase/driver/redshift.clj

```diff
@@ -49,6 +51,30 @@
   [& args]
   (apply (get-method driver/describe-table :sql-jdbc) args))
 
+(defmethod sql-jdbc.sync/describe-fks-sql :redshift
```

Comment:
See the implementation of redshift's JDBC driver for reference
https://github.com/aws/amazon-redshift-jdbc-driver/blob/master/src/main/java/com/amazon/redshift/jdbc/RedshiftDatabaseMetaData.java#L3974